### ISSUE TITLE
mlDatabaseReplicaDataDirectories does not split string into Map<String,List<String>>

### ISSUE THREAD

**Issue by ewillia1983** [marklogic-community/ml-gradle#489]
It looks like there is a difference in how the mlDatabaseDataDirectories and mlDatabaseReplicaDataDirectories properties are handled.  I was expecting them to be the same.

I am automating a large cluster, with each node having 4 volumes attached.  I am dedicating volume 1 and 2 as primary forest volumes, and volumes 3 and 4 as replica forest volumes.  The primaries are working as expected, but the replicas are not.

Here are screenshots from debugging the issue:

![image](https://user-images.githubusercontent.com/31934733/61552897-0f616b80-aa27-11e9-8a13-7b331b0d9bc2.png)

![image](https://user-images.githubusercontent.com/31934733/61552862-ee991600-aa26-11e9-80f0-0a0e73604a8e.png)

build.gradle setup:

```
plugins {
  id 'java-library'
  id "net.saliman.properties" version "1.5.1"
  id "com.marklogic.ml-gradle" version "3.15.1"
}

dependencies {
  compile group: 'com.marklogic', name: 'ml-app-deployer', version: '3.15.0'
}

repositories {
  jcenter()
}
```

**Comment by rjrudin** [comment]
Just to clarify - you're looking to specify multiple data directories for replicas for a particular database?

**Comment by ewillia1983** [comment]
That's correct, primary forests would go on /var/opt/volume1 and /var/opt/volume2; replica forests would go on /var/opt/volume3 and /var/opt/volume4.  Here's the link to the documentation I'm building this around.  

[https://docs.marklogic.com/guide/ec2.pdf](https://docs.marklogic.com/guide/ec2.pdf) -- page 10

`Each instance of MarkLogic Server should be configured with a maximum of 5 Hi-IO volumes/forests.`

Thinking about this more, I may have opened this bug in the wrong project.  This is from the underlying ML-App-Deployer project.  Please let me know if I should move it over!

**Comment by rjrudin** [comment]
Got it, and no problem opening issues here - I move them as needed, which I'll do for this one.
